starter-code/routes/auth/login.js

```diff
@@ -0,0 +1,68 @@
+var express = require('express');
+var router = express.Router();
+
+// User model
+const User           = require("../../models/user");
+
+// Bcrypt to encrypt passwords
+const bcrypt         = require("bcrypt");
+const bcryptSalt     = 10;
+
+router.get("/login", (req, res, next) => {
+  res.render("auth/login");
+});
+
+// Redirect to login
+router.get("/auth", (req, res) => {
+  res.redirect("/login");
+});
+
+router.post("/login", (req, res, next) => {
+  const username = req.body.username;
+  const password = req.body.password;
+
+  if (username === "" || password === "") {
+    res.render("auth/login", {
+      errorMessage: "Indicate a username and a password to log in"
+    });
+    return;
+  }
+
+  User.findOne({ "username": username },
+    "_id username  following",
+    (err, user) => {
+      if (err || !user) {
+        res.render("auth/login", {
+          errorMessage: "The username doesn't exist"
+        });
+        return;
+      } else {
+        if (bcrypt.compareSync(password, user.password)) {
+          //saves login in the session
+          req.session.currentUser = user;
+          // When User logged in 
+          res.redirect("/home");
```

Comment:
You can't redirect to an .ejs file. You have to redirect to a GET.

In you routes you should have a index file with:
```
/* GET home page. */

router.get('/', (req, res) => {
   if (!req.session.currentUser) {
     res.redirect('/login');         // if you have no user session, it redirects to get login
   } else {
     res.render('home', {user: req.session.currentUser});  // if you have user, it renders home passing the user json
   }
});
```
Then, you should redirect to this get